src/api/history.ts

```diff
@@ -0,0 +1,73 @@
+import { HistoryEntry } from '../components/landing/pages/history/history'
+import { expectResponseCode, getBackendUrl } from '../utils/apiUtils'
+
+export const getHistory = async (): Promise<HistoryEntry[]> => {
+  const response = await fetch(getBackendUrl() + '/history')
+  expectResponseCode(response)
+  return await response.json() as Promise<HistoryEntry[]>
+}
+
+export const setHistory = async (entries: HistoryEntry[]): Promise<void> => {
+  const response = await fetch(getBackendUrl() + '/history', {
+    method: 'POST',
+    mode: 'cors',
+    cache: 'no-cache',
+    credentials: 'same-origin',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    redirect: 'follow',
```

Comment:
Why should we follow redirects?

Reply:
Because.. of.. redirecting load balancers? 😅
No srsly. Why shouldn't we?

Reply:
To be honest I just copied the settings of the other api calls, maybe we should extract default fetch config

Reply:
extracted